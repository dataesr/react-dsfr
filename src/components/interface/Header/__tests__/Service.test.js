import renderer from 'react-test-renderer';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import {
  Service,
} from '../index';

Enzyme.configure({ adapter: new Adapter() });

describe('<Service />', () => {
  it('renders correctly', () => {
    const component = renderer
      .create(
        <Service
          title="title"
          description="description service"
        />,
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
