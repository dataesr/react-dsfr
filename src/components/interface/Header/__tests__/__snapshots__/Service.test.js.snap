// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Service /> renders correctly 1`] = `
<div
  className="fr-header__service"
>
  <a
    className="fr-header__service-title"
    target="_self"
    title="title"
  >
    title
  </a>
  <p
    className="fr-header__service-tagline"
  >
    description service
  </p>
</div>
`;
